### PR TITLE
Execute automated tests in headless mode by default

### LINKED ISSUE #803: Automated tests should run in headless mode by default
## Description
As decided in the meeting, by default the automated tests should run in headless mode.

## Mocks
N/A

### Update [Required]
- [ ] Modify tests to run in headless mode.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
30 mins.

### PR DESCRIPTION
**Pull request by Monal5031**
# Description
This PR modifies the existing automated tests to be executed in headless mode by default.

Fixes #803 

# Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
Pull this branch locally and run
```
python manage.py test -v 2
```
you can see that now the browser won't open when tests start.

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: vms/administrator/tests/test_report.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
 import json
 
 from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+from selenium.common.exceptions import NoSuchElementException
```
Comment (naman1901) [inline vms/administrator/tests/test_report.py]: This import required? And where? Because the other files don't have this import.

Comment (Monal5031) [reply vms/administrator/tests/test_report.py]: This was already imported earlier. I've just rearranged it.

**Comment by Monal5031** [comment]
@naman1901 @MehaKaushik @ayushgarg1804 @pavankm Please review.

**Comment by Monal5031** [comment]
@ayushgarg1804  Yeah sure I'll give it a try, would make sense to have it.

**Pull request merged**